tests/framework/installer/ceph_manifests.go

```diff
@@ -353,6 +359,32 @@ parameters:
 	return sc
 }
 
+func (m *CephManifestsMaster) GetNFSStorageClass(fsName, nfsClusterName, server, storageClassName string) string {
+	// Create a CSI driver storage class
+	csiCephFSNodeSecret := "rook-csi-cephfs-node"               //nolint:gosec // We safely suppress gosec in tests file
+	csiCephFSProvisionerSecret := "rook-csi-cephfs-provisioner" //nolint:gosec // We safely suppress gosec in tests file
+	sc := `
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ` + storageClassName + `
+provisioner: ` + m.settings.OperatorNamespace + `.nfs.csi.ceph.com
+parameters:
+  clusterID: ` + m.settings.Namespace + `
+  fsName: ` + fsName + `
+  nfsCluster: ` + nfsClusterName + `
+  server: ` + server + `
+  pool: ` + fsName + `-data0
+  csi.storage.k8s.io/provisioner-secret-name: ` + csiCephFSProvisionerSecret + `
+  csi.storage.k8s.io/provisioner-secret-namespace: ` + m.settings.Namespace + `
+  csi.storage.k8s.io/node-stage-secret-name: ` + csiCephFSNodeSecret + `
+  csi.storage.k8s.io/node-stage-secret-namespace: ` + m.settings.Namespace + `
+mountOptions:
+  - "nolock"
```

Comment:
Is there a reason why we're supplying "nolock" here? A comment about why we do so would be good for future reference.